apps/rush-lib/assets/rush-init/common/config/rush/version-policies.json

```diff
@@ -77,7 +77,13 @@
      * inappropriately.  The minor/patch version parts will be bumped independently according
      * to the types of changes made to each project, according to the "rush change" command.
      */
-    "lockedMajor": 3
+    "lockedMajor": 3,
+
+    /**
+     * (Optional) If false, developers of the specified version policy will no longer
+     * be prompted for changelog files during the rush change command.
```

Comment:
Incorrect wording:  `rush change` collects "change log entries" and makes "change files".  Then `rush version` integrates "change files" to produce a "change log" (i.e. CHANGELOG.md).

Reply:
Maybe something like this:

"(Optional) When publishing is managed by Rush, by default the "rush change" command will request changes for any projects that are modified by a pull request.  These change entries will produce a CHANGELOG.md file.  If you author your CHANGELOG.md manually or announce updates in some other way, set "noChangeLogRequired" to true to tell "rush change" to ignore the projects belonging to this version policy"